packages/studio-ui/src/hooks/useInjectActiveStyles.tsx

```diff
@@ -10,12 +10,12 @@ import { StudioStore } from "../store/models/StudioStore";
  * styling of the current active page and component tree.  Styling is cleared
  * from the iframe before each injection.
  */
-export default function useInjectUserStyles(
+export default function useInjectActiveStyles(
   iframeDocument: Document | undefined
 ) {
-  const [userStyleImports, loadedStyleFilepaths] = useStudioStore(
+  const [activeStylesFilepaths, loadedStyleFilepaths] = useStudioStore(
```

Comment:
nit: It's a little odd to me that one of these variable names has "styles" plural and the other has "style". Can we pick one? I'm slightly leaning toward "\<blah\>StyleFilepaths" so there's not as many plurals

Reply:
changed to the one with less plurals!